src/Components/ImagesTable/ImagesTable.scss

```diff
@@ -0,0 +1,19 @@
+.pf-c-table {
```

Comment:
I checked the pixel distances on the mock-up and did this to get it pretty close, but that was actually something I was hoping to discuss when the code was reviewed... when it comes to padding, how close to the mock-up should we aim to be?

Reply:
I'm not 100% sure all the mockups are pixel-perfect. Plus we don't do pixel testing (which is a bigger project that would be neat to do at some point). 

For now visually close + not too much css/trying to rely on standard patternfly margin/padding variables is what I've gone with.

Reply:
Thank you for the insights. In the interest of minimizing CSS I went ahead and removed the top padding modification, so it is 1.5 rem (PF4 default) now instead of 1 rem. After spending a few days away from this PR and coming back to it, the default padding now looks fine to me...